### DISCUSSION TITLE
Ambient temperature while charging

### DISCUSSION THREAD

**Discussion by Jadhav-Sourabh** [pybamm-team/PyBaMM#2163] [General]
is it possible to see the variation of ambient temperature while charging the cell?

**Comment by brosaplanella** [comment]
Do you mean ambient temperature or cell temperature?

**Reply by Jadhav-Sourabh** [reply]
ambient temperature

**Reply by brosaplanella** [reply]
You can plot it using the standard methods (see [this notebook](https://colab.research.google.com/github/pybamm-team/PyBaMM/blob/main/examples/notebooks/Getting%20Started/Tutorial%203%20-%20Basic%20plotting.ipynb) for more info). One of the variables is `"Ambient temperature [K]"`.

**Reply by Jadhav-Sourabh** [reply]
Can we vary the ambient temperature throughout charging process? small variation like from 15 celsius to 20 celsius.

**Reply by brosaplanella** [reply]
Yes. You can add an ambient temperature that changes with time. It should follow similarly to changing the input current (see [this notebook](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/notebooks/parameterization/change-input-current.ipynb), the section "Adding your own current function" might be helpful).

**Reply by Jadhav-Sourabh** [reply]
Thank you for help

**Reply by rtimms** [reply]
[This example](https://github.com/pybamm-team/PyBaMM/blob/develop/examples/scripts/DFN_ambient_temperature.py) does exactly that.
